packages/ui-modal/src/Modal/index.tsx

```diff
@@ -160,23 +160,25 @@ class Modal extends Component<ModalProps, ModalState> {
   renderChildren() {
     const { children, variant, overflow } = this.props
 
-    return Children.map(
-      children as (HeaderChild | BodyChild | FooterChild)[],
-      (child) => {
-        if (!child) return // ignore null, falsy children
-
-        if (matchComponentTypes<BodyChild>(child, [ModalBody])) {
-          return safeCloneElement(child, {
-            variant: variant,
-            overflow: child.props.overflow || overflow
-          })
-        } else {
-          return safeCloneElement(child, {
-            variant: variant
-          })
-        }
+    return Children.map(children, (child) => {
+      if (!child) return // ignore null, falsy children
+
+      if (matchComponentTypes<BodyChild>(child, [ModalBody])) {
```

Comment:
will this part work with custom children? I mean the `variant` and `overflow` props? or do the users have to add these manually too?

Reply:
No, they only work if they used our own children components.